### ISSUE TITLE
Replace aacdecoder lib with Exoplayer on android

### ISSUE THREAD

**Issue by tlenclos** [tlenclos/react-native-audio-streaming#12]
As discussed here https://github.com/tlenclos/react-native-audio-streaming/issues/5#issuecomment-237539525

Exoplayer is a better, recent, updated lib to handle media.
Example implementation https://github.com/xeodou/react-native-player

**Comment by tlenclos** [comment]
I tried ExoPlayer and it's working nice except and allow much more control on the stream. 

Unfortunately it does not support shoutcast metadata, somebody tried to implement it here https://github.com/google/ExoPlayer/issues/466#issuecomment-103476038 but it's outdated and we need to apply this patch on the current version of ExoPlayer (and maybe create a PR).

**Comment by brettpappas** [comment]
Are you still looking at replacing aacdecoder with ExoPlayer? is the only holdup getting shoutcast metadata support for ExoPlayer?

**Comment by tlenclos** [comment]
Yes I still think this is the way to go, and the only blocker is shoutcast metadata support.
